cas-server-webapp-support/src/main/java/org/jasig/cas/web/report/InternalConfigStateController.java

```diff
@@ -0,0 +1,254 @@
+    }
+
+    private static final class BeanObjectJsonSerializer extends AbstractJacksonBackedJsonSerializer<Object> {
+        private static final long serialVersionUID = 691461175315322624L;
+
+        /**
+         * Instantiates a new Bean object json serializer.
+         */
+        public BeanObjectJsonSerializer() {
+            super(new MinimalPrettyPrinter());
+        }
+
+        @Override
+        protected ObjectMapper initializeObjectMapper() {
+            final ObjectMapper mapper = super.initializeObjectMapper();
+
+            final FilterProvider filters = new SimpleFilterProvider()
+                    .addFilter("beanObjectFilter", new CasSimpleBeanObjectFilter());
+            mapper.setFilters(filters);
+
+            mapper.setVisibility(PropertyAccessor.ALL, JsonAutoDetect.Visibility.ANY);
+            mapper.configure(SerializationFeature.WRITE_EMPTY_JSON_ARRAYS, false);
+            mapper.configure(SerializationFeature.WRITE_NULL_MAP_VALUES, false);
+            mapper.setSerializationInclusion(JsonInclude.Include.NON_EMPTY);
+            mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+            mapper.addMixIn(Object.class, CasSimpleBeanObjectFilter.class);
+            mapper.disable(SerializationFeature.INDENT_OUTPUT);
+            mapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+
+            return mapper;
+        }
+
+        @Override
+        protected Class<Object> getTypeToSerialize() {
+            return Object.class;
+        }
+
+        @JsonFilter("beanObjectFilter")
+        private static class CasSimpleBeanObjectFilter extends SimpleBeanPropertyFilter {
+            private static final Logger LOGGER = getLogger(CasSimpleBeanObjectFilter.class);
+
+            private static final String[] EXCLUDE_FIELDS = new String[] {"logger"};
+
+            @Override
+            public void serializeAsField(final Object pojo, final JsonGenerator jgen,
+                                         final SerializerProvider provider,
+                                         final PropertyWriter writer) throws Exception {
+                try {
+                    if (!canSerializeField(pojo, writer)) {
+                        return;
+                    }
+                    super.serializeAsField(pojo, jgen, provider, writer);
+                } catch (final Exception e) {
+                    LOGGER.debug(e.getMessage());
+                }
+            }
+
+            /**
+             * Can serialize field?
+             *
+             * @param pojo the pojo
+             * @param writer the writer
+             * @return the boolean
+             */
+            private boolean canSerializeField(final Object pojo, final PropertyWriter writer) {
+                boolean foundPackage = false;
+                final String packageName = pojo.getClass().getPackage().getName();
+                for (int i = 0; !foundPackage && i < INCLUDE_PACKAGES.length; i++) {
```

Comment:
It's not, because while beans under org.jasig are retrieved, those beans themselves could have dependencies to other components that may not be under org.jasig. (i.e. a bean could have ApplicationContext, etc as part of Spring) So we really are interested in the stuff that we have written :)